test/integration/Client/client_lib_multi_argobots_test.cpp

```diff
@@ -40,7 +40,7 @@ void thread_function(void*tt)
     chronicle_attrs.emplace("TieringPolicy", "Hot");
     ret = client->CreateChronicle(chronicle_name, chronicle_attrs, flags);
     std::cout << "tid=" << t->tid << " Createchronicle {" << chronicle_name << "} ret: " << ret << std::endl;
-    assert(ret == CL_SUCCESS || ret == CL_ERR_CHRONICLE_EXISTS || ret == CL_ERR_NO_KEEPERS);
+    assert(ret == chronolog::CL_SUCCESS || ret == chronolog::CL_ERR_CHRONICLE_EXISTS || ret == chronolog::CL_ERR_NO_KEEPERS);
```

Comment:
This line is longer than 120. Please reformat it. Please do the same for all other test case files.